README.md

```diff
@@ -2,45 +2,60 @@
 
 Nextflow plugin for interacting with [Quilt](https://quiltdata.com/) packages.
 
-`nf-quilt` currently allows you to publish the outputs of a workflow run as a Quilt package. WHen you launch a pipeline with the `nf-quilt` plugin, it will collect every output file that is published to S3 and publish a Quilt package upon workflow completion.
+`nf-quilt` currently allows you to publish the outputs of a workflow run as a Quilt package. WHen you launch a pipeline with the `nf-quilt` plugin, it will publish a Quilt package upon workflow completion that contains output files published to S3.
 
-## Requirements
+## Getting Started
 
-- Nextflow 22.04 or later
-- Python 3.7 or later
-- The [quilt-api](https://github.com/seqeralabs/quilt-api) package
+To use the `nf-quilt` plugin, you need Nextflow 22.04 (or later) and Python 3.7 (or later).
 
-## Usage
-
-To enable the plugin, add the following snippet to your `nextflow.config`:
+Install the [quilt-api](https://github.com/seqeralabs/quilt-api) package:
+```bash
+pip3 install git+ssh://git@github.com/seqeralabs/quilt-api.git
```

Comment:
In the previous line it mentions `https://github.com/..` here instead is `git+ssh://git@github.com`. The first is better because does not required SSH keys

Reply:
The ssh syntax is there because the quilt-api repo is private. If you want we can just make it public like nf-quilt, I just thought it was going to be an xpack or something.

Reply:
There's a real need to keep in a separate repo? Could not be merged into this? The ideal solution would be being able to install the python deps on-fly

Reply:
I just followed the example of the graphviz DAG renderer, if Nextflow can't find graphviz on your system then it tells you to go install it.

We can probably host the plugin code and python code in the same repo if you want, but I don't think we should try to install python packages at runtime. It would be error-prone.

Reply:
Let's start moving the python code here, then. 

Concerning the self-install, the less the user has to do the better.

Reply:
Why don't we handle this part in a separate PR

Reply:
Sure